nextflow.config

```diff
@@ -23,22 +23,26 @@ params {
     contrast_pairs = 'DEFAULT2'
     genelist = 'NO_FILE'
     relevel = 'NO_FILE2'
-    species = null
     batch_effect = null
-    kegg_blacklist = 'NO_FILE3'
     quote = 'NO_FILE4'
     min_DEG_pathway = 1
-    vst_genes_number = 1000
     use_vst = false
+    vst_genes_number = 1000
     skip_pathway_analysis = false
+    input_type = 'featurecounts'
```

Comment:
other params that you introduced are missing here. E.g. `keytype`, `organism`... Add with default of `null`

Reply:
Yep, still produces a `Missing `fromPath` parameter` error

Reply:
ah right I remember now!